### PR TITLE
TC-OPSTATE-2.5,6: Fix messages, re-establish subscription after reset

### PR DESCRIPTION
**Pull request by cecille**
Changes the prompts on OPSTATE-2.5 and 2.6 so the operator can see what to do. 

Also adds a subscription re-establishment step after the dut is reset.

### CODE REVIEW

**Review by tcarmelveilleux — APPROVED** [review]

**Review by saurabhst — APPROVED** [review]

File: src/python_testing/TC_OpstateCommon.py [hunk 1]
```diff
@@ -1116,7 +1120,14 @@ async def TEST_TC_OPSTATE_BASE_2_5(self, endpoint=1):
             # reboot outside CI environment.
             if not self.is_ci:
                 self.wait_for_user_input(prompt_msg="Restart DUT. Press Enter when ready.\n")
+                # Expire the session and re-establish the subscription
                 self.default_controller.ExpireSessions(self.dut_node_id)
+                if self.check_pics(f"{self.test_info.pics_code}.S.E01"):
+                    # Subscribe to Events and when they are sent push them to a queue for checking later
```
Comment (plauric) [inline src/python_testing/TC_OpstateCommon.py:1126]:
```suggestion
                    # Subscribe to Events and when they are received sent push them to a queue for checking later
```

Is this what you meant to type?

Comment (plauric) [inline src/python_testing/TC_OpstateCommon.py:1126]:
```suggestion
                    # Subscribe to Events and when they are received push them to a queue for checking later
```

File: src/python_testing/TC_OpstateCommon.py [hunk 2]
```diff
@@ -1259,11 +1270,12 @@ async def TEST_TC_OPSTATE_BASE_2_6(self, endpoint=1):
...
         await sub_handler.start(self.default_controller, self.dut_node_id, endpoint)
 
+        self.step(3)
         if self.pics_guard(self.check_pics(f"{self.test_info.pics_code}.S.M.ST_RUNNING")):
-            self.step(3)
             self.send_manual_or_pipe_command(name="OperationalStateChange",
                                              device=self.device,
-                                             operation="Start")
+                                             operation="Start",
+                                             msg="Put the DUT into the running state")
```
Comment (plauric) [inline src/python_testing/TC_OpstateCommon.py:1278]:
```suggestion
                                             msg="Put DUT in running state")
```

To be consistent with the similar messages below.

Comment (plauric) [inline src/python_testing/TC_OpstateCommon.py:1278]: Question about the statement that follows below (i.e. time.sleep(1)): should we add a PIXIT code to control these waits between issuing a command to the DUT and reading some attribute to check the effect? For this TC and for many others. The "real DUTs", that the vendors will attempt to certify, may have wild variations for the response times.

Not something to be fixed in this PR, obviously, but if you agree, or have other ideas, would you mind creating an issue for this?

**Review by plauric — APPROVED** [review]

**Review by jmeg-sfy — APPROVED** [review]

File: src/python_testing/TC_OpstateCommon.py [hunk 3]
```diff
@@ -1311,11 +1321,12 @@ async def TEST_TC_OPSTATE_BASE_2_6(self, endpoint=1):
             self.skip_step(7)
 
         sub_handler.reset()
+        self.step(8)
```
Comment (rbultman) [inline src/python_testing/TC_OpstateCommon.py:1324]: The else portion of this if/else block should be removed since the step was moved here.

**Review by rbultman — CHANGES_REQUESTED** [review]

File: src/python_testing/TC_OpstateCommon.py (hunk 2 above)
Comment (cecille) [reply src/python_testing/TC_OpstateCommon.py]: I think it's fine - the test is just attempting to verify that the device isn't over reporting on the Q attribute - 10 s should be fine to assess that.

File: src/python_testing/TC_OpstateCommon.py (hunk 3 above)
Comment (cecille) [reply src/python_testing/TC_OpstateCommon.py:1324]: thanks - removed. PTAL.

**Review by rbultman — APPROVED** [review]

**Pull request merged**